### ISSUE TITLE
Need filter for host in our RNA Seq analysis page

### ISSUE THREAD

**Issue by ARWattam** [PATRIC3/patric3_website#1645]
A request from our favorite user, Andrew Warren

**Comment by aswarren** [comment]
@ARWattam Did I provide any more detail than that in my request?

**Comment by mshukla1** [comment]
@aswarren 

I interpret this as limiting the selection of reference genomes to only 10 host genomes when user selects HiSAT2 as the strategy. Can you make the change in the submission ui? 

<img width="350" alt="Screen Shot 2019-04-13 at 10 54 02 AM" src="https://user-images.githubusercontent.com/1236463/56082141-d0de6400-5dda-11e9-8359-3340e131a468.png">

**Comment by ARWattam** [comment]
No, aswarren did not provide any additional comments when originally posted, but I believe that he wanted there to be a check box for the eukaryotic genomes.  I think it would be very difficult for them to find them through the filter without that.
